### ISSUE TITLE
docs(aio): Testing: Description and code don't match

### ISSUE THREAD

**Issue by andys8** [angular/angular#16064]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
The code example is a single line using `subscribe`:
```js
this.route.params.subscribe(p => this.getHero(p && p['id']));
```
The description talks about the usage of `pluck` and `forEach`. They aren't used (probably anymore) in the code. This is misleading.

> The expression after route.params chains an Observable operator that plucks the id from the params and then chains a forEach operator to subscribe to id-changing events. The id changes every time the user navigates to a different hero.
> The forEach passes the new id value to the component's getHero method (not shown) which fetches a hero and sets the component's hero property. If the id parameter is missing, the pluck operator fails and the catch treats failure as a request to edit a new hero.

**Screenshot**

![image](https://cloud.githubusercontent.com/assets/13085980/25103744/f837d788-23bd-11e7-84b7-cbed22e7c693.png)

**Expected behavior**
The description should match the code. The description should be replaced or removed.

**Affected Files**
https://github.com/angular/angular/blob/master/aio/content/examples/testing/src/app/hero/hero-detail.component.ts
https://raw.githubusercontent.com/angular/angular/master/aio/content/guide/testing.md

**Comment by kapunahelewong** [comment]
@jenniferfell This issue might be resolved in #20697.

**Comment by jenniferfell** [comment]
Closing. Doc and example were updated at some point to use paramMap.subscribe instead of the code above:

ngOnInit(): void {
  // get hero when `id` param changes
  this.route.paramMap.subscribe(pmap => this.getHero(pmap.get('id')));
}

filename=https://angular.io/guide/testing
